staff_features/resources/resource_container_instance.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Resource Container Instance
+  Background:
+    Given an administrator user is logged in
+      And a Resource has been created
+      And the Resource is opened in edit mode
+  Scenario: Add container instance
+     When the user clicks on 'Instances'
+      And the user clicks on 'Add Container Instance'
+      And the user selects 'Accession' from 'Type' in the 'Instances' form
+      And the user clicks on the first dropdown in the "Instances" form
+      And the user clicks on "Create" in the dropdown menu in the "Instances" form
```

Comment:
And the user clicks on 'Create' in the dropdown menu in the 'Top Container'  in the 'Instances' form